### ISSUE TITLE
Map tiles not loading

### ISSUE THREAD

**Issue by Lineflyer** [cgeo/WhereYouGo#64]
**Describe the bug:**
Using the beta version 2020.03.05 no map tiles are loading for me independently which only provider I choose.

**To Reproduce:**
1. Start WhereYouGo
2. Select map
3. Choose between the various sources in Menu - Map type 

**Actual behavior/state after performing these steps:**
None of those I tried show any tiles on my device. Empty (white) map.

**Expected behavior/state after performing these steps:**
Map tiles should be shown.
Additionally I guess, that some tile providers are no longer available/valid (e.g. OpenCycleMap is no longer available for free). But however I guess Mapnik should work like it does in c:geo.

I wasn't able to check the logs yet.

**Version of WhereYouGo used:**
2020.03.05

**Is the problem reproducible:**
Yes 

**Additional context:**
Samsung S10, Android 10

**Comment by Lineflyer** [comment]
This is how it looks on my Samsung S10, Android 10 (not working):
![Screenshot_20200306-134457](https://user-images.githubusercontent.com/949669/76084779-ff47cd80-5fb0-11ea-8e31-04d1ce756e22.jpg)

In order to take a look into traces I connected my Samsung S4, Android 5.01.1, but on this device its working. Do you notice the different menu bar? The 5.0.1 shows less menu items (in fact no settings, etc. which are shown on the Android 10 device behind the three-dot).
![Screenshot_2020-03-06-13-44-18](https://user-images.githubusercontent.com/949669/76084878-2a322180-5fb1-11ea-98cd-34274d594cb0.png)

**Comment by Lineflyer** [comment]
I tried to use an v3 mapsforge offline map file instead of online maps on the problematic Android 10 device. This works normal.
So the mapsforge function is working in general, but not loading online tiles.

**Comment by Lineflyer** [comment]
Here we go:
Samsung S10, Android 10

```
03-06 14:14:40.464 18882 29140 E TileDownloader: java.io.IOException: Cleartext HTTP traffic to a.tile.openstreetmap.org not permitted
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at com.android.okhttp.HttpHandler$CleartextURLFilter.checkURLPermitted(HttpHandler.java:124)
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.execute(HttpURLConnectionImpl.java:480)
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getResponse(HttpURLConnectionImpl.java:429)
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at com.android.okhttp.internal.huc.HttpURLConnectionImpl.getInputStream(HttpURLConnectionImpl.java:252)
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at org.mapsforge.android.maps.mapgenerator.tiledownloader.TileDownloader.executeJob(TileDownloader.java:65)
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at org.mapsforge.android.maps.mapgenerator.MapWorker.doWork(MapWorker.java:97)
03-06 14:14:40.464 18882 29140 E TileDownloader: 	at org.mapsforge.android.maps.PausableThread.run(PausableThread.java:141)
03-06 14:14:40.464 18882 29140 I System.out: (HTTPLog)-Static: isSBSettingEnabled false
03-06 14:14:40.464 18882 29140 E TileDownloader: null
```

**Comment by FriedrichFroebel** [comment]
This seems to be related to #45 in some way.

**Comment by bekuno** [comment]
This is now the issue which will be solved with #45.

The old addresses are http. They must be migrated to https.
Some of them do not work anymore or have a new license model or other complications.
Maybe new sources (cyclosm.org, öpnvkarte.de, ...) can be integrated.
